client/flags/flags.go

```diff
@@ -136,8 +136,8 @@ func AddTxFlagsToCmd(cmd *cobra.Command) {
 	f.Bool(FlagOffline, false, "Offline mode (does not allow any online functionality)")
 	f.BoolP(FlagSkipConfirmation, "y", false, "Skip tx broadcasting prompt confirmation")
 	f.String(FlagSignMode, "", "Choose sign mode (direct|amino-json|direct-aux|textual), this is an advanced feature")
-	f.Uint64(FlagTimeoutHeight, 0, "Set a block timeout height to prevent the tx from being committed past a certain height")
-	f.Bool(FlagUnordered, false, "Enable unordered transaction delivery; must be used in conjunction with --timeout-height")
+	f.Int64(FlagTimeoutTimestamp, 0, "Set a block timeout timestamp to prevent the tx from being committed past a certain time")
```

Comment:
any reason this became an Int instead of uint?

Reply:
`time.Unix(sec, nsec int64)` takes int64s https://github.com/cosmos/cosmos-sdk/blob/35d3ff99641192eee40f774a1291492fc50f4c54/client/tx/factory.go#L91-L92